common/changes/sparo/feat-support_add_profile_in_checkout_2024-02-26-04-10.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "sparo",
+      "comment": "refactor sparo checkout profile logic",
```

Comment:
Rephrase this to describe what benefit it provides for the end user (e.g. `Improve efficiency of the \"sparo checkout\" command` or `Fix handling of some edge cases for \"sparo checkout\"`).  If there is no observable benefit for end users, then don't document the change.